test/functional/test_framework/script.py

```diff
@@ -371,11 +371,12 @@ def __init__(self, d=0):
 
     @staticmethod
     def encode(obj):
+        val = obj.value
```

Comment:
Could discard these unrelated these changes.

Reply:
it reduces the verbosity of it, I can remove but seemed better to me.

Reply:
I understand but IMO it's better without.

Reply:
already removed

Reply:
Yeah I know :)